### ISSUE TITLE
Pydio does not check for dom extension but requires it

### ISSUE THREAD

**Issue by cbj4074** [pydio/pydio-core#1388]
Until dom (and by extension, phpxml) are installed and enabled in PHP, attempts to navigate to the Pydio document root fail with the following error:

Fatal error:  Uncaught Error: Class 'DOMDocument' not found in /var/www/pydio/pydio-core/core/src/core/src/pydio/Core/PluginFramework/Plugin.php:443

If the `DOMDocument` class is required, it would be great to see the requirement noted in the installation docs. Even better would be to check the dependency programatically during pre-installation checks (of which there are several already).

**Comment by cdujeu** [comment]
hi ben, what's your php version?

**Comment by cbj4074** [comment]
Thanks for the quick reply here, Charles!

I'm running a four different versions in a VM, and I can't remember whether I was using 7.0 or 7.1 at that time, but I'm pretty sure that it was 7.0. The fix in my case was to `apt-get install php7.0-xml php7.1-xml`, but I didn't test both versions independently.

Just let me know if I can answer any other questions!

**Comment by cdujeu** [comment]
mmm, just coming back on that one : the strange part is that there is already such a test https://github.com/pydio/pydio-core/blob/develop/core/src/core/src/pydio/Tests/PHPDomXML.php since ... 2010 :-)
Was it a clear install or an upgrade? Which packaging?

**Comment by cbj4074** [comment]
Hey there @cdujeu , sorry to leave you hanging!

I ended-up not using Pydio for the project I was planning at the time I encountered this issue, so I never circled-back in this regard.

It was a clear install, and I'm quite sure that I simply followed the instructions from https://pydio.com/en/docs/v8/install-archive-0 .

If you were never able to reproduce the issue, then I think it makes sense to leave it closed. I'm too mired in other efforts at the moment to give this the attention that it deserves.

Thank you, and my apologies for the late reply!
